src/main/java/dansplugins/factionsystem/utils/Locale.java

```diff
@@ -1,13 +1,29 @@
-package dansplugins.factionsystem.utils;
-
-import dansplugins.factionsystem.services.LocalLocaleService;
-
-/**
- * @author Daniel McCoy Stephenson
- */
-public class Locale {
-    
-    public static String get(String key) {
-        return LocalLocaleService.getInstance().getText(key);
-    }
-}
+package dansplugins.factionsystem.utils;
+
+import java.util.HashMap;
+
+import dansplugins.factionsystem.services.LocalLocaleService;
+
+/**
+ * @author Daniel McCoy Stephenson
+ */
+public class Locale {
+
+    private static HashMap<String, Integer> keyUses = new HashMap<>();
+    
+    public static String get(String key) {
+        recordKeyUse(key);
+        return LocalLocaleService.getInstance().getText(key);
+    }
+
+    static void recordKeyUse(String key) {

```

Comment:
I don't think this is necessary as it also partly depends on the user :thinking:

Reply:
If we track key usage and make it persistent, we'll be able to cycle through the keys loaded into the locale service and see which ones have 0 uses. Then we can created a report that lists the keys not being used and we can remove them from the .tsv files. There might be a better way of going about this, but I think this is a way forward.

Reply:
then you can add metrics to make it easier to get data because if you put it here, only the person who gives the log will know

Reply:
I'm not familiar with Metrics. Would you be able to elaborate on this? @VoChiDanh

Reply:
I haven't had time to research this yet

Reply:
No worries!